config/crd/bases/ndb.nutanix.com_databases.yaml

```diff
@@ -119,6 +119,7 @@ spec:
                       associated with this database
                     properties:
                       dailySnapshotTime:
+                        default: "12:34:56"
```

Comment:
it is better to have a default snapshot time in the middle of the night when a small DB downtime is easier to tolerate.

Reply:
Sure! Changed it!